spec/classes/shared_base.rb

```diff
@@ -11,14 +11,26 @@
   end
 end
 
+shared_examples_for :base_8 do
+  include_context :base
+
+  it do
+    is_expected.to contain_yumrepo('epel').with(
+        mirrorlist: 'https://mirrors.fedoraproject.org/metalink?repo=epel-8&arch=$basearch',
```

Comment:
You have an extra two spaces of indent here.

Reply:
This is indented exactly the same way as the other sections. Have you looked at this in context?

Reply:
Fixed.